raiden/utils/upgrades.py

```diff
@@ -48,14 +56,21 @@ def get_db_version(db_filename: Path):
         detect_types=sqlite3.PARSE_DECLTYPES,
     )
     cursor = conn.cursor()
+
     try:
-        cursor.execute('SELECT value FROM settings WHERE name=?;', ('version',))
-        query = cursor.fetchall()
-        if len(query) == 0:
-            return 0
-        return int(query[0][0])
+        cursor.execute('SELECT value FROM settings WHERE name="version";')
+        result = cursor.fetchone()
     except sqlite3.OperationalError:
-        return 0
+        raise RuntimeError(
```

Comment:
Here I tightened up the checks, because we know the database exists I think it should be expected to be a valid Raiden db.